### PR TITLE
Fix handle server errors

### PR DESCRIPTION
**Pull request by lomamech**
- Sanitaze Html delete </p> not open in using
- Using lib/errors/NotFound

@sfount

### CODE REVIEW

**Comment by jniles** [comment]
Nice commit messages!

**Comment by sfount** [comment]
@lomamech Agreed @jniles. The commit messages are very helpful when reviewing what the code changes do.

File: client/test/e2e/services/services.spec.js [hunk 1]
```diff
@@ -18,7 +18,7 @@ describe('Services Module', function () {
   var DEFAULT_SERVICE = 4;
   var SERVICE_RANK = helpers.random(DEFAULT_SERVICE);
 
-  var DELETE_SUCCESS = 4;
+  var DELETE_SUCCESS = 5;
```
Comment (sfount) [inline client/test/e2e/services/services.spec.js]: @lomamech My E2E test fails with this new value of 5 but it passes with 4. This test should be updated to ensure it passes.

**Comment by lomamech** [comment]
@sfount 
Fix error in test E2E for services

**Comment by sfount** [comment]
Overall a very good re-factor. 
- [x] Passes integration tests
- [x] Passes end to end test 
- [x] Code reviewed (no blocking issues)

**Pull request merged**